pkg/registry/registry.go

```diff
@@ -201,27 +205,30 @@ func (r *Registry) registryHandler(rw mux.ResponseWriter, req *http.Request) {
 			cacheType = "miss"
 		}
 		metrics.MirrorRequestsTotal.WithLabelValues(registryName, cacheType, sourceType).Inc()
-		return
+		return "mirror"
 	}
 
 	// Serve registry endpoints.
 	if dgst == "" {
 		dgst, err = r.ociClient.Resolve(req.Context(), ref)
 		if err != nil {
 			rw.WriteError(http.StatusNotFound, err)
-			return
+			return ""
```

Comment:
A bit of a nitpick, but from the user's perspective this is actually `manifest` or `blob`. If it turns out that `refType` is "cucumber" it would be better if we ended up on line 231 below than complaining that the "ref" (which is unlikely to be a ref) is unknown.

Reply:
While this is true the handler is not as important as the error message is included in the error log. I will think about this quickly to see if it is worth making the distinction.

Reply:
I will move this code to the manifest handler later which should resolve this.